containers/images/redis/README.md

```diff
@@ -1,74 +1,33 @@
-# Redis packaged by SourceMation
+# Redis
```

Comment:
The previous README.md was better, in my opinion. Does the new Redis not work in the same way?